### ISSUE TITLE
Change value before submitting.

### ISSUE THREAD

**Issue by sankhadeeproy007** [erikras/redux-form#2248]
Hi, I'm sure this is a very silly question. I just want to know how to modify the value of a field before I submit. I'm handling a price field and I want to change the `,` to `.` before sending it to the backend . I believe normalizing is the way to go, but I still can't figure out how to do it. Also I checked the documentation, there were some [actions ](http://redux-form.com/6.3.0/docs/api/Props.md/) I could call to do the required thing. But I'm not sure how to do it. Any help?

**Comment by erikras** [comment]
You have several options. If you want the value _displayed_ with the `,`, you could set up a [`parse` and `format` pair](http://redux-form.com/6.3.1/docs/ValueLifecycle.md/) to always keep the value the way you want it in the Redux store.

Or, possibly more simply, you could just modify it as you are submitting it.

```jsx
onSubmit = (values) =>
  ajax.post('/saveStuff', {
    ...values,
    price: fixComma(values.price)
  })
```

Does that help?

**Comment by sankhadeeproy007** [comment]
Yes lovely! Thanks.
